apollo-router/src/plugin/mod.rs

```diff
@@ -482,6 +470,8 @@ pub trait PluginUnstable: Send + Sync + 'static {
     #[allow(private_interfaces)]
     fn connector_service(
         &self,
+        _subgraph_name: &str,
+        _source_name: &str,
         _service_name: &str,
```

Comment:
if you could document what's the meaning of these differents params that would be useful. And an example for each for example ?